extension/js/common/composer.ts

```diff
@@ -1641,7 +1646,15 @@ export class Composer {
       }
       this.renderSenderAliasesOptionsToggle();
     } else {
-      $('.close_new_message').click(Ui.event.handle(() => this.app.closeMsg(), this.getErrHandlers(`close message`)));
+      $('.close_new_message').click(Ui.event.handle(async () => {
+        if (!this.isSendMessageInProgress) {
```

Comment:
Looks good, could be simplified as:
```js
if (!this.isSendMessageInProgress || await Ui.modal.confirm('A message is currently being sent. Closing the compose window may abort sending the message.\nAbort sending?')) {
  this.app.closeMsg();
}
```

Reply:
I've changed it. 
I didn't do it because I a bit don't like long lines but if it's okay for you I will try to write the code this way.